docs/tech-reference/yum-plugins.rst

```diff
@@ -589,6 +589,12 @@ Optional Configuration Parameters
  a repository publish.  If unspecified it will not run due to the extra time needed to
  perform this operation.
 
+``repoview``
+ Boolean flag to indicate whether or not static HTML files should be generated by
+ the repoview tool during a repository publish to improve browsing experience. If
+ specified it will also set a ``generate_sqlite`` flag to ``True``, because sqlite
```

Comment:
"a ``generate_sqlite``" should be "the ``generate_sqlite``" and no comma after ``True``, I believe